.travis.yml

```diff
@@ -3,23 +3,19 @@ language: cpp
 
 env:
   global:
-    - LLVM_VERSION=3.8.0
     # Maintenance note: to move to a new version
     # of boost, update both BOOST_ROOT and BOOST_URL.
     # Note that for simplicity, BOOST_ROOT's final
     # namepart must match the folder name internal
     # to boost's .tar.gz.
-    - LCOV_ROOT=$HOME/lcov
-    - GDB_ROOT=$HOME/gdb
-    - BOOST_ROOT=$HOME/boost_1_60_0
-    - BOOST_URL='http://sourceforge.net/projects/boost/files/boost/1.60.0/boost_1_60_0.tar.gz'
-
-# Travis is timing out on Trusty.  So, for now, use Precise.  July 2017
-dist: precise
+    - BOOST_ROOT=$HOME/boost_1_65_1
+    - BOOST_URL='http://sourceforge.net/projects/boost/files/boost/1.65.1/boost_1_65_1.tar.gz'
```

Comment:
I'd like to start getting boost from bintray instead of sourceforge (sourceforge has been shady in the past). For 1_65_1 is would be https://dl.bintray.com/boostorg/release/1.65.1/source//boost_1_65_1.tar.bz2

Reply:
Any objection to https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.gz so I don't have to change the `tar` command downstream?

Reply:
Nope, that works for me.